Add Mailwip extension

xmok opened the pull request:
## Description

This is a Raycast extension for [Mailwip](https://mailwip.com/) (previously known as **Hanami**). With this extension you can Batch Delete Domains, Manage Aliases, and View MailLog in Mailwip.

## Screencast

https://github.com/raycast/extensions/assets/30526133/a650f424-443e-4930-8ca1-f3d3e285fd88

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

mil3na commented on extensions/mailwip/src/batch-delete-domains.tsx:
```diff
@@ -0,0 +1,82 @@
+        </ActionPanel>
+      }
+    >
+      <Form.Description text="Batch Delete Domains (you will have to manually remove these from Extension Preferences)" />
```
This is not the best UX. Maybe there would be another way to manage the domains instead of leaving them on the preferences?

![CleanShot 2023-07-04 at 15 49 19@2x](https://github.com/raycast/extensions/assets/611002/28a334ed-a5c3-4657-b145-463de0396128)

xmok replied:
Hi Milena:

Thank you for the detailed review. I agree that this is not the best UX so I will experiment replacing Preferences with a useCachedState() approach and push an update when done.

mil3na reviewed (commented):
Hi! Thank you so much for your contribution to Raycast!
My name is Milena and I will be reviewing your extension today :)

Your extension seems to be working fine according to my tests. I just left a few recommendation comments :)

xmok commented:
https://github.com/raycast/extensions/assets/30526133/04ce59e5-14c5-4888-a995-8c081101ac0b

Here's a video of the changes to the domains. There are a few breaking changes:

- renamed "batch-delete-domains" to "domains"
- removed the "domains" Preference

mil3na reviewed (approved):
LGTM!
